companion/src/storage/categorized.cpp

```diff
@@ -54,10 +56,20 @@ bool CategorizedStorageFormat::load(RadioData & radioData)
         categoryIndex++;
       }
       else {
-        qDebug() << "Loading" << line;
+        QByteArray curline;
+        if (line.contains(FIELD_DELIMITER)) {
+          modelIndex=line.mid(0,line.indexOf(FIELD_DELIMITER)).toInt();
+          curline = line.mid(line.indexOf(FIELD_DELIMITER)+1);
+          qDebug() << "Detected" << curline;
+        }
+        else {
+          curline = line.data();
+        }
+
```

Comment:
Qstring curline = QString(curline).trimmed()  to remove all the potential extra white whitespaces